src/backend/marsha/core/tests/test_lti_utils.py

```diff
@@ -2522,12 +2522,11 @@ def _test_lti_get_resource_other_pl_site_not_portable_instructor(
         passport = factories.ConsumerSiteLTIPassportFactory(
             consumer_site__domain="example.com"
         )
-        one_not_portable = random.choice([True, False])
         resource = factory(
             id="77fbf317-3e99-41bd-819c-130531313139",
             playlist__lti_id="a-playlist",
-            playlist__is_portable_to_playlist=one_not_portable,
-            playlist__is_portable_to_consumer_site=not one_not_portable,
+            playlist__is_portable_to_playlist=False,
+            playlist__is_portable_to_consumer_site=False,
```

Comment:
Given the name "one_not_portable" I think the intent was more:
```suggestion
            playlist__is_portable_to_playlist=one_not_portable,
            playlist__is_portable_to_consumer_site=not one_not_portable,
```

Reply:
This is the original code which fails.

Reply:
Yeah I read the docstring again and what you did corresponds to the docstring. So :+1: